relay-event-normalization/src/normalize/span/description/mod.rs

```diff
@@ -532,6 +562,43 @@ fn scrub_function(string: &str) -> Option<String> {
     Some(FUNCTION_NORMALIZER_REGEX.replace_all(string, "*").into())
 }
 
+fn scrub_mongodb_query(query: &str, command: &str, collection: &str) -> Option<String> {
+    let original: Value = match serde_json::from_str(query) {
+        Ok(v) => v,
+        Err(_) => return None,
+    };
+
+    let mut scrubbed_map: Map<String, Value> = Map::new();
+    original.as_object()?.iter().for_each(|(key, value)| {
```

Comment:
`for_each` is a bit of a [rust anti pattern](https://rust-lang.github.io/rust-clippy/master/index.html#/needless_for_each), you can just use a `for` here.